### ISSUE TITLE
regression(1.04) `nim doc main.nim` generates broken html (no css)

### ISSUE THREAD

**Issue by timotheecour** [nim-lang/Nim#12998]
`nim doc main.nim` generates broken html (css file absent)
regression possibly introduced in #12816

### Example
real life example: https://disruptek.github.io/gittyup/gittyup.html
minimal example:
```nim
proc fun*()=
  ## some proc
  discard
```

### Current Output
![image](https://user-images.githubusercontent.com/2194784/71625296-e3ccda80-2b9b-11ea-8c91-c4b04069be67.png)

### Expected Output
as in nim 1.0.4:
![image](https://user-images.githubusercontent.com/2194784/71625279-c7c93900-2b9b-11ea-9bb8-793dbb6e278b.png)

### Possible Solution
* either embed the css in html file or update instructions in `nim --fullhelp` for `nim doc`

### Additional Information
D20191231T070930
chrome console shows:
```
nimdoc.css:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
dochack.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
```
it used to show:
```
dochack.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
```

* working in version 1.0.4
* broken in b67dea7a33d24cd54afa8bd37677d066f4be0da1

**Comment by juancarlospaco** [comment]
`nim rst2html` also broken https://github.com/nim-lang/Nim/issues/12980#issue-543123277

**Comment by yewpad** [comment]
![grafik](https://user-images.githubusercontent.com/54741912/71850093-cd5edb80-30d3-11ea-9499-3a102a1aa12a.png)
Confirming broken doc gen with latest compiler build (as of ba4fbb63285ce198a9feb62e8058889462a96a87).

**Comment by timotheecour** [comment]
@yewpad the regression was likely introduced in  #12816 , if you want to fix you can look at that PR :-)

**Comment by yewpad** [comment]
So I got it working again. I had to put the CSS back into the `nimdoc.cfg` file so that the doc gen can actually include the css in the html file.

Problem is that @araq wanted to have the CSS outside of the `nimdoc.cfg` file for either maintenance purposes or because of the tests failing when the CSS gets changed.

I can't exactly tell you why the doc command doesn't copy over the `nimdoc.css` file into the directory where the docs are being placed in but my guess is that the `copyFile` call only gets executed when `koch docs` is executed? Idk, it's hard to figure out. The doc gen implementation is so messed up and spaghetti-like that I didn't care anymore and put the css back in the config file.

Also, I don't consider the fix done just yet. I have to discuss with @araq whether or not it is acceptable to have the CSS back in `nimdoc.css`. I think for now it will be fine but we'll see.

![grafik](https://user-images.githubusercontent.com/54741912/71855400-3862df00-30e1-11ea-9187-74b122a27145.png)
